pyat/at/physics/fastring.py

```diff
@@ -41,12 +39,9 @@ def _rearrange(ring, split_inds=[]):
 
 def _fring(ring, split_inds=[], detuning_elem=None):
     all_rings, merged_ring = _rearrange(ring, split_inds=split_inds)
-    ibegs = get_refpts(merged_ring, 'xbeg')
-    iends = get_refpts(merged_ring, 'xend')
-    markers = numpy.sort(numpy.concatenate((ibegs, iends)))
-    with warnings.catch_warnings():
```

Comment:
I think this change is fine, however we generally use fond_orbit_sync for this kind of calculation without radiation. This is why I had suppressed the warning.

Reply:
Agreed, but in this case, `dct=0` is strictly equivalent to `dp=0` : it's the on-momentum orbit. And it's the default behaviour.